plugins/chain_interface/include/eosio/chain/plugin_interface.hpp

```diff
@@ -18,20 +18,15 @@ namespace eosio { namespace chain { namespace plugin_interface {
    };
 
    struct runtime_metric {
-      const metric_type type;
-      const std::string family;
-      const std::string label;
-      std::atomic<int64_t> value;
+      const metric_type type = metric_type::gauge;
+      const std::string family = "";
+      const std::string label = "";
```

Comment:
These strings are copied over and over, but small string optimization should avoid dynamic memory allocation. We should strive to always keep these strings under 24 chars.